<launch>

  <include file="$(find usb_cam)/launch/usb_cam_multi.launch" />

    <node ns="node1" pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager"/>
    <node ns="node2" pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager"/>  
    
    <!-- detect aruco markers -->
    <node ns="node1" pkg="nodelet" clear_params="true" type="nodelet" name="aruco_detect" args="load aruco_pose/aruco_detect nodelet_manager">
        <remap from="image_raw" to="/node1/usb_cam/image_rect"/>
        <remap from="camera_info" to="/node1/usb_cam/camera_info"/>
        <param name="length" value="0.7"/>
    </node>

    <node ns="node2" pkg="nodelet" clear_params="true" type="nodelet" name="aruco_detect" args="load aruco_pose/aruco_detect nodelet_manager">
        <remap from="image_raw" to="/node2/usb_cam/image_rect"/>
        <remap from="camera_info" to="/node2/usb_cam/camera_info"/>
        <param name="length" value="0.7"/>
    </node>


</launch>
